### ISSUE TITLE
Downloads doesnt have details

### ISSUE THREAD

**Issue by mosi3883** [jeffhill/Dota2#349]
### Description

its like 2 years, Downloads Section doesnt have Heros Icon and match duration
All it says(No Match Details...)

### Screenshots

![image](https://user-images.githubusercontent.com/54463682/171815467-e1f1e3db-2601-432c-8a61-1c3ca38d4e09.png)

**Comment by pseudoparenchymatous** [comment]
They should put hero names/icons, match length, player names, and put the winner and team kills with toggle on/of spoiler. And if spoiler is on they should show the end of match page where you can see the graphs, etc.

**Comment by Soong45** [comment]
To help narrow down the time frame when this stopped working, I have a replay from 23rd April 2020, where it shows what the expected behaviour should be.
![Bug_illustration](https://user-images.githubusercontent.com/36033864/171852270-0a63c471-f27e-441a-a83e-3ee8c5679193.png)

**Comment by org30h** [comment]
To add on this, the "saved to watch later" which is supposed to show something like recent downloads/rep marked as watch later, is not implemented and left blank.

![image](https://user-images.githubusercontent.com/10111038/174500970-00bb1560-44ff-4191-9fd1-e02b113c60c9.png)

**Comment by Soong45** [comment]
> To add on this, the "saved to watch later" which is supposed to show something like recent downloads/rep marked as watch later, is not implemented and left blank.

![Saved_for_later](https://user-images.githubusercontent.com/36033864/174581271-9ae04d3c-f3eb-4cf3-97e4-99db5098ebf4.png)
This does work and shows replays where you've asked the client to "Notify when replay is ready". (Although sometimes it won't appear until after you have restarted the client.)
If there are any issues with this, I think it should be a separate ticket to this one.

**Comment by HsDL-Galand** [comment]
> To help narrow down the time frame when this stopped working, I have a replay from 23rd April 2020, where it shows what the expected behaviour should be. I can provide the replay file if it'd be helpful. You can also click on the match to go to the game page, which works for the one displaying the hero portraits, but does not for the "No match details" replays. ![Bug_illustration](https://user-images.githubusercontent.com/36033864/171852270-0a63c471-f27e-441a-a83e-3ee8c5679193.png)

I think I've found the exact day that this bug occurred, which was between 20 September 2020 and 21 September 2020. As you can see in my screenshot, I have downloaded one game on 20/09, and another game on 21/09; but only the first one shows match details.
In Italian "Nessun dettaglio sulla partita" means "no match details"

I REALLY hope this detail helps to find the solution to this; one last thing, can somebody tell me why is it so difficult to fix this?

![Dota 2 11_07_2022 00_14_24](https://user-images.githubusercontent.com/59095670/178164083-4ce67cb9-e663-4888-82a5-e29913d85dca.png)
t

**Comment by mosi3883** [comment]
> I REALLY hope this detail helps to find the solution to this; one last thing, can somebody tell me why is it so difficult to fix this?

It's not about difficulty,it's just not that important

**Comment by HsDL-Galand** [comment]
> ![replays](https://user-images.githubusercontent.com/106764092/194073554-69e3fe18-2070-4d40-a26a-dee68be747b2.png)
> 
> Seems to be fixed?

YES IT IS. Finally after 2 whole years of this bug, we can watch the match details again, thanks Valve. (this screenshot represents my actual situation, compared to the other screenshot I sent few messages above)

![Dota 2 05_10_2022 15_41_55](https://user-images.githubusercontent.com/59095670/194075323-84f74171-5d02-448c-8ec9-76f49474a624.png)
